Jenkinsfile

```diff
@@ -1,107 +1 @@
-// This Jenkinsfile's main purpose is to show a real-world-ish example
-// of what Pipeline config syntax actually looks like.
-pipeline {
-    // Make sure that the tools we need are installed and on the path.
-    tools {
-        maven "mvn"
-        jdk "jdk8"
-    }
-
-    agent none
-
-    // Set log rotation, timeout and timestamps in the console
-    options {
-        buildDiscarder(logRotator(numToKeepStr:'10'))
-        timestamps()
-        timeout(time: 120, unit: 'MINUTES')
-    }
-
-    // Make sure we have GIT_COMMITTER_NAME and GIT_COMMITTER_EMAIL set due to machine weirdness.
-    environment {
-        GIT_COMMITTER_NAME = "jenkins"
-        GIT_COMMITTER_EMAIL = "jenkins@jenkins.io"
-        NEWER_CORE_VERSION = "2.164.3"
-        TEST_TIMEOUT = "600"
-    }
-
-
-    stages {
-        // While there is only one stage here, you can specify as many stages as you like!
-        stage("build") {
-            parallel {
-                stage("linux") {
-                    agent {
-                        label "highmem"
-                    }
-                    steps {
-                        sh "mvn -B clean install -Dmaven.test.failure.ignore=true -Djenkins.test.timeout=${TEST_TIMEOUT}"
```

Comment:
TBD whether this still needs to be overridden. Are there individual tests which normally run >10m?!

Reply:
Indeed there are, though I doubt the previous configuration actually affected the timeout: passing a system property to the `mvn` process does not automatically cause it to be propagated to the Surefire booter process.

Reply:
> passing a system property to the `mvn` process does not automatically cause it to be propagated to the Surefire booter process

Correction: it does, but only if it is passed directly from the command line using `-D`; an entry in `<properties>` is _not_ propagated.